tests/integration/icontracts/contracts/read_erc20.py

```
@@ -1,3 +1,4 @@
+# v0.1.0
 # { "Depends": "py-genlayer:test" }
 
 from genlayer import *
```

Ruff (0.11.9)

`from genlayer import *` used; unable to detect undefined names

(F403)